includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -1259,7 +1259,7 @@ public function process_refund( $order_id, $amount = null, $reason = '' ) {
 				$response = WC_Stripe_API::request( $request, 'refunds' );
 			}
 		} catch ( WC_Stripe_Exception $e ) {
-			WC_Stripe_Logger::log( 'Error: ' . $e->getMessage() );
+			WC_Stripe_Logger::error( 'Error: ' . $e->getMessage() );
```

Comment:
As above RE some exceptions including the full response object contents. We can consider improving the content in this PR or a follow-up.
```suggestion
			WC_Stripe_Logger::error( 'Exception processing refund', [ 'exception' => $e->getMessage() ] );
```